Diz.Core/SampleRomData.cs

```diff
@@ -1,11 +1,17 @@
-﻿using Diz.Core.model;
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using Diz.Core.model;
 using Diz.Core.util;
 using IX.Observable;
 
 namespace Diz.Core
 {
     public class SampleRomData : Data
     {
+        public static string GetSampleUtf8CartridgeTitle() => "｢ﾎ｣ abcｦｧｨ TEST123"; // don't pad here
```

Comment:
these are all 1-byte ShiftJIS glyphs, try something with a 2-byte character make sure all our stuff holds up.

in particular anywhere in the code we're comparing a string length to the 21 byte length, we should replace to compare it to 21 BYTES not 21 CHARS

Reply:
for this PR, I think we're good enough at the moment. I made a followup issue for later, lower pri: https://github.com/Dotsarecool/DiztinGUIsh/issues/51